src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -42,31 +42,19 @@ public class TestConnectionCommand : PSCmdlet
         /// <summary>
         /// Force using IPv4 protocol.
         /// </summary>
-        [Parameter(ParameterSetName = DefaultPingSet)]
```

Comment:
Which possibilities exist that would be agnostic to IPv4/v6?

Reply:
I don't understand you question. I mean that later it would be easier to add new features in the cmdlet (like PathPing). (I am thinking about enhanced version of the cmslet with remoting support: user in A point could ping/traceroute/... C from B)

Reply:
Thanks!

> This would have to be supported by PSRemoting anyway

It seems we have a tracking issue to have remoting common parameters for all cmdlets.
With this we could get functionality like Test-NetConnection.

Reply:
>  It seems the win32 APIs for ICMP echo actually support a similar parameter,

It is WMI.